default/empire_statistics.txt

```diff
@@ -1,3 +1,11 @@
+
+Statistic name = "MILITARY_STRENGTH_STAT" value =
+    Sum value = (LocalCandidate.Attack * (LocalCandidate.Structure + 2 * LocalCandidate.MaxShield))
+        condition = And [
```

Comment:
The condition is aligned with the value, these being the two parameters for the Sum, so I don't think it should be dedented.  It could be you were thrown off by the wraparound that the github display has.